canopen/pdo/base.py

```diff
@@ -528,11 +528,11 @@ def wait_for_reception(self, timeout: float = 10) -> float:
         return self.timestamp if self.is_received else None
 
 
-class Variable(variable.Variable):
+class PdoVariable(variable.Variable):
```

Comment:
This is pure bikeshedding, but why do you use CamelCase with the lowercase `Pdo` abbreviation here, while for the OD you use the all-caps `OD` spelling as in `ODVariable`?  Shoult consistenly use uppercase for abbreviations here.

Reply:
@acolomb you're right, this is inconsistent. I didn't make a very deliberate choice, rather trying to adopt the gist from the rest of canopen. After looking more closer at it, I see canopen is equally inconsistent, e.g. `class PDO` vs `class PdoBase`. Looking at all class names, the majority is CamelCase, with the notable exception of `PDO`, `TPDO`, `RPDO` and not used with an additional suffix.

To be consistent with naming in canopen, `ODVariable` should then be `OdVariable` perhaps?

Reply:
Yes, I guess that's an acceptable solution.  Just don't introduce even more inconsistency ;-)

Reply:
Naming is probably inconsistent everywhere else anyway, so it's not super important. Not sure what's best for readability, but I'm nowadays leaning more towards keeping abbreviations upper-case even if it makes it harder to separate the words.